### DISCUSSION TITLE
Reshade support or a way to have both them run at the same time?

### DISCUSSION THREAD

**Discussion by YamiN0Neko** [Friendly0Fire/GW2Radial#303] [Ideas]
Can you support reshade because since the game is now using the dx11 the dll file that reshade creates is a dxgi.dll, which is the same as the radial moving forward so I was wondering if you could some how use  a different file name that isn't dxgi,dll or support reshade to have both of them running. radial's dll has to go next to the exe and same for reshade dll since I  tried to have either one in the  bin64 folder and they don't get pick up so can you please change your file name to something different since chainloading isn't being supported. Will be great to be able to  run reshade  and radial at the same time. This are the only two addons I run and I cannot use them anymore since dx9 is not going to bee supported anymore.

**Comment by Friendly0Fire** [comment]
Use https://github.com/megai2/SelectRenderer.

**Reply by Kironcete** [reply]
could you elaborate on this? I don't really know how to use it nor did I find any readme that explains it. I have the same problem as the poster. How can I use SelectRenderer to use at the same time reshade and radial when both have a dxgi.dll which are different (so if I install one then the other, it asks me to replace the old one).

**Reply by Friendly0Fire** [reply]
The other addon has step by step instructions on how to set it up, just follow them to the letter and you'll be able to select Gshade/Reshade in the GUI.

**Reply by Kironcete** [reply]
> The other addon has step by step instructions on how to set it up, just follow them to the letter and you'll be able to select Gshade/Reshade in the GUI.

Yea well i don't see those instructions on the main selecrender webpage. All i see is a path to where install the reshade dxgi.dll (which I did), but not how to install selectrender, maybe I can't find it but I really did check :S I can download the files, but I can't not make selecrender to work cause I don't know how to install it :S

**Reply by Friendly0Fire** [reply]
It's in the addon manager, you install it exactly like you with for Radial.

**Reply by Kironcete** [reply]
> It's in the addon manager, you install it exactly like you with for Radial.

Yea, I downloaded addonmanger, installed selecrenderer (in fact if I open adonmanager now, it says it has latest selecrender version installed). But when I open the game, no UI shows so it is not working. I don't know, but thanks T_T

**Reply by Friendly0Fire** [reply]
You'd have to open an issue in the SelectRenderer repo then, unfortunately I cannot help you beyond that.
